modelina-website/src/components/playground/options/JavaGeneratorOptions.tsx

```diff
@@ -115,8 +123,15 @@ class JavaGeneratorOptions extends React.Component<
             />
           </label>
         </li>
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Include Jackson serialization :">
+            <p>
+            Jackson annotations are a set of annotations provided by the Jackson library, which is a popular Java-based JSON processing library. These annotations are used to configure and control how Java objects are serialized to JSON and deserialized from JSON.
+            <br/><br/>
+            Annotations in Java are represented by the @ symbol followed by the annotation name. 
```

Comment:
This is very generic wording about what toString is, however it does not explain what it does to the code generation.

Reply:
fixed!! :tada: